docker/templates/includes/_footer.html

```diff
@@ -20,6 +20,19 @@
             <i class="fab fa-github fa-2x"></i>
           </a>
       </li>
+
+      <li class="list-inline-item">
+          <a href="https://www.youtube.com/channel/UChv_8AbbznYS-xa256nXV6w" target="_blank">
+            <i class="fab fa-youtube fa-2x"></i>
+          </a>
+      </li>
+		  
+      <li class="list-inline-item">
+          <a href="https://tinyurl.com/yan5yzp9" target="_blank">
```

Comment:
the endpoint should be: devopsloft.slack.com?

Reply:
this is the same tinyurl we are using for months.
it works for me:
when I use the url from Chrome where I usually use Slack, I get into #general directly
when I use it in other browser, I get into "Join the Slack workspace DevOps Loft" page

Reply:
lets change it devopsloft.slack.com

That should work. right?

Reply:
I changed it manually to devopsloft.slack.com and it worked the same as the tinyurl.

In addition, just want to let you know that I faced the same issue with the CRLF line endings, had to change it to LF

Reply:
Than, maybe just add information on how to get into the Slack workspace.

@ShironB thanks for updating. @Balex3000 is working on the PR which should address this issue.

Reply:
changed the link to slack and retested on dev